shared/AppInsightsCore/src/JavaScriptSDK/SenderPostManager.ts

```diff
@@ -57,7 +57,7 @@ export class SenderPostManager {
 
         dynamicProto(SenderPostManager, this, (_self, _base) => {
 
-            let _sendCredentials = true; // for 1ds
+            let _sendCredentials = true; // default
```

Comment:
I believe that this class is used by both AI and 1DS, so I don't think its going to be a simple true/false